backend/src/polling_station/structs.rs

```diff
@@ -23,7 +23,7 @@ pub struct PollingStation {
     #[schema(nullable = false)]
     pub polling_station_type: Option<PollingStationType>,
     pub address: String,
-    pub postal_code: String,
+    pub postal_code: Option<String>,
```

Comment:
I don't think we need to make this an `Option`, because strings can be left empty and we currently already use the postal code field that way in the frontend.